features/support/pages/app.rb

```diff
@@ -7,7 +9,12 @@
 # adapter that delegates most of it's behavior to the appropriate pages, instead of manually wrapping it as it
 # does now.
 class App
-  attr_accessor :logged_in_user
+  PAGES = {
+    admin_moderator_actions_page: Admin::ModeratorActionsPage,
+    admin_pending_projects_page: Admin::PendingProjectsPage
+  }.freeze
```

Comment:
@bhaibel - Thoughts on using a pattern like this so we're not always defining `x_page` methods everywhere?